kuma/static/styles/minimalist/print.scss

```diff
@@ -0,0 +1,30 @@
+@import '../includes/vars';
+@import './mixins/print-mixins';
+
+header,
+.notification-tray,
+.global-notice,
+.nav-footer,
+.contributors,
+.crumbs,
+.breadcrumbs,
+a.edit-section,
+.page-buttons,
+.open-in-host-container,
+.newsletter-box,
+#page-attachments,
+#offline-dialog-content,
+#wiki-left {
+    display: none;
+}
```

Comment:
And, with everything in the same print stylesheet, we can have this tiny optimization :)